pkg/provider/azure_storageaccount.go

```diff
@@ -206,12 +207,17 @@ func (az *Cloud) EnsureStorageAccount(accountOptions *AccountOptions, genAccount
 			klog.V(2).Infof("azure - no matching account found, begin to create a new account %s in resource group %s, location: %s, accountType: %s, accountKind: %s, tags: %+v",
 				accountName, resourceGroup, location, accountType, kind, accountOptions.Tags)
 
+			largeFileShareState := storage.LargeFileSharesStateDisabled
+			if accountOptions.EnableLargeFileShare {
+				largeFileShareState = storage.LargeFileSharesStateEnabled
+			}
 			cp := storage.AccountCreateParameters{
 				Sku:  &storage.Sku{Name: storage.SkuName(accountType)},
 				Kind: kind,
 				AccountPropertiesCreateParameters: &storage.AccountPropertiesCreateParameters{
 					EnableHTTPSTrafficOnly: &enableHTTPSTrafficOnly,
 					NetworkRuleSet:         networkRuleSet,
+					LargeFileSharesState:   largeFileShareState,
```

Comment:
this may change the default behavior of `LargeFileSharesState`, is it possible that 
```
if accountOptions.EnableLargeFileShare {
   cp.AccountPropertiesCreateParameters.LargeFileSharesState = storage.LargeFileSharesStateEnabled
}

```